workflow/rocoto/workflow_tasks.py

```diff
@@ -543,6 +549,52 @@ def aeroanlfinal(self):
 
         return task
 
+    def landanlinit(self):
+
+        dump_suffix = self._base["DUMP_SUFFIX"]
+        dmpdir = self._base["DMPDIR"]
+        atm_hist_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_HISTORY_TMPL"], {'RUN': 'gdas'})
+        dump_path = self._template_to_rocoto_cycstring(self._base["COM_OBSDMP_TMPL"],
+                                                       {'DMPDIR': dmpdir, 'DUMP_SUFFIX': dump_suffix})
```

Comment:
You are adding a dependency on COM_OBSDMP, but the script is using COM_OBS. I think you should probably be using COM_OBS_TMPL here instead.
```suggestion
        dump_path = self._template_to_rocoto_cycstring(self._base["COM_OBS_TMPL"] , {'RUN': self.cdump})
```

Reply:
For our DA cycling tests, we use `DMPDIR` to define for OBS data in the `config.base`. Therefore, we use `COM_OBSDMP_TMPL` here, which is the same as `ATM DA` and `AERO DA`. @CoryMartin-NOAA Am I correct? Thanks.

Reply:
@jiaruidong2017 I think the preferred way going forward @aerorahul correct me if I'm wrong, is that the prep job copies/links obs to COM_OBS and then our DA jobs grab them from there. Thus this should have the same dependency as the atmanlinit or aeroanlinit jobs.

Reply:
@CoryMartin-NOAA is correct.

Reply:
Removed the dependency on `COM_OBSDMP_TMPL`, and use `COM_OBS` for obs. Thanks @CoryMartin-NOAA